sled-agent/src/instance_manager.rs

```diff
@@ -20,11 +22,6 @@ use std::collections::BTreeMap;
 use std::sync::{Arc, Mutex};
 use uuid::Uuid;
 
-#[cfg(not(test))]
-use crate::instance::Instance;
-#[cfg(test)]
-use crate::instance::MockInstance as Instance;
-
```

Comment:
I'm ripping off this band-aid a little early, but these mock-based tests are terrible (see: https://github.com/oxidecomputer/omicron/issues/2422 ) and rather than making them work with this U.2 integration, I got rid of some of them.